src/main/java/seedu/recipe/ui/events/EditRecipeEvent.java

```diff
@@ -16,47 +14,21 @@ public class EditRecipeEvent extends Event {
      */
     public static final EventType<EditRecipeEvent> EDIT_RECIPE_EVENT_TYPE = new EventType<>(Event.ANY, "EDIT_RECIPE");
```

Comment:
While we're at it i'll delete `AddRecipeEvent` and push because it is no longer in use.

Reply:
fixed, removed